dev/benchmarks/macrobenchmarks/macos/Flutter/Flutter-Debug.xcconfig

```diff
@@ -0,0 +1,2 @@
+#include "Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig"
+#include "ephemeral/Flutter-Generated.xcconfig"
```

Comment:
I don't really know how desktop works so I might be asking dumb things. What's the `macos/Flutter/ephemeral/flutter_export_environment.sh` file for? I don't see it referenced anywhere.

Reply:
I think I just never special cased that code to not generate the script for macOS.  It would (someday) be used for macOS add-to-app though.

Reply:
ah ok. good to know. I was trying to figure out where it plugged in :D